templates/http-php/metadata/spin-template.toml

```
@@ -0,0 +1,8 @@
+manifest_version = "1"
+id = "http-php"
+description = "HTTP PHP environment"
```

For other templates like this we've used the form "HTTP request handler using [language]"
```suggestion
description = "HTTP request handler using PHP"
```